libmscore/cmd.cpp

```diff
@@ -183,8 +183,8 @@ void Score::endCmd(bool rollback)
       undoStack()->endMacro(noUndo);
 
       if (dirty()) {
-            masterScore()->_playlistDirty = true;  // TODO: flag individual operations
-            masterScore()->_autosaveDirty = true;
+            masterScore()->setPlaylistDirty();  // TODO: flag individual operations
```

Comment:
This line (as well as other replacements of `_playlistDirty = true` to `setPlaylistDirty()`) should be the one that can address many of the issues not fixed by the previous patch versions. However the issue still remains when undoing/redoing certain operations (like undoing of deleting of a segno marking), so probably a similar line should be added to `undoRedo()`. Otherwise lots of individual operations like certain elements adding/removing should also call `setPlaylistDirty()`, and adding all those calls may take a much larger effort.

Reply:
See #5073